benchmarks/templates/benchmarks/landing_page.html

```diff
@@ -1,11 +1,165 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>Placeholder. Actual Page Coming Soon</title>
-</head>
-<body>
-
-<h1>Placeholder. Actual Page Coming Soon</h1>
-</body>
-</html>
\ No newline at end of file
+{% load static %}
```

Comment:
So far all pages extended the `base.html` page so that global site tags etc are loaded in a unified manner. This is currently lost on the landing page. I think you should have the `landing_page.html` extend the `base.html`, you'll save a lot of redundancy. Note that you can still change the navbar -- using the same code you have now -- because the `block` replaces the corresponding block in `base.html`